### PR TITLE
feature/CD-130: Table component

### PR DESCRIPTION
**Pull request by Gerasimov94**

### CODE REVIEW

File: src/modules/Packages/PackagesRoot/components/PackagesRoot.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import Flex from 'common/components/Flex/Flex';
+
+import PackagesTableContainer from 'modules/Packages/PackagesTable/containers/PackagesTableContainer';
+
+export default class PackagesRoot extends PureComponent {
+	public render() {
+		return (
+			<Flex>
+				<Flex>
+					ebanina sleva
```
Comment (gusartus) [inline src/modules/Packages/PackagesRoot/components/PackagesRoot.tsx]: 😯😯😯

File: src/utils/formatBytes.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
+import memoize from 'lodash.memoize';
+import i18n from 'locales/i18nextConfig';
```
Comment (temihren) [inline src/utils/formatBytes.ts]: screenshot from 2019-02-18 11-43-51

Comment (gusartus) [inline src/utils/formatBytes.ts:4]: screenshot from 2019-02-18 11-44-30

File: src/utils/formatBytes.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	if (typeof numberVar !== 'number' || !Number.isFinite(numberVar)) {
+		throw new Error('Must be a number.');
+	}
+
+	const unitArr = getBytesUnits();
+	const maxUnitIndex = unitArr.findIndex(([unit]) => unit === maxUnit);
+
+	let count = 0;
+	while (numberVar >= 1024 && count < maxUnitIndex) {
+		numberVar = numberVar / 1024;
```
Comment (gusartus) [inline src/utils/formatBytes.ts:32]: [tslint] Reassigning parameter 'numberVar' is forbidden. (no-parameter-reassignment)

**Comment by gusartus** [comment]
By clicking on NEXT STEP button in /user/create
screenshot from 2019-02-18 11-52-27

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**